frontend/src/components/common/authorized-menu/common.ts

```diff
@@ -25,6 +20,11 @@ const routes: NavigationMenuItem[] = [
         iconName: 'education',
         href: AppRoute.EDUCATION,
       },
+      {
+        name: 'Interview',
+        iconName: 'education',
```

Comment:
So now both education and interview nav links will have education icon?

Reply:
According to the Figma, yes